lttoolbox/tmx_compiler.cc

```diff
@@ -536,16 +520,16 @@ TMXCompiler::align(vector<int> &origin, vector<int> &meta)
           if(vectorcmp(origin, numbers_origin_start[j],
                        meta, i, nl))
           {
-            modified_meta.push_back(L'@');
-            modified_meta.push_back(L'(');
-            wchar_t *valor = new wchar_t[8];
-            swprintf(valor, 8, L"%d", j+1);
-            for(int k = 0, limit3 = wcslen(valor); k != limit3; k++)
+            modified_meta.push_back('@');
+            modified_meta.push_back('(');
+            UChar* valor = new UChar[8];
+            u_snprintf(valor, 8, "%d", j+1);
+            for(int k = 0, limit3 = u_strlen(valor); k != limit3; k++)
```

Comment:
Pretty sure `u_snprintf()` return the number of written units, making u_strlen() superfluous. Not that it matters much for such a tiny string.